service_ui/src/FAQ.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FAQsPageProps } from './FAQsPage';
+
+function FAQ ({faq, index, toggleFAQ}:{faq:any;index:number,toggleFAQ:any}) {
```

Comment:
i would generally prefer to keep this the same shape as all other tsx, in which we have a specific prop interface, and we write the component as "const" and not "function"
if you have time, it would be nice to change it, but you don't have to.
and if you would like, I set up a snippet shortcut for myself that can already build a template of that (cause I always have to copy-paste it from somewhere since I never remember what it's supposed to be like), and I can set up the same for you XD